src/cookbooks/active_dau.md

```diff
@@ -24,6 +24,21 @@ ORDER BY
     1 ASC
 ```
 
+[`clients_last_seen`](../datasets/bigquery/clients_last_seen/reference.md) can also be used, but is only available in BigQuery. Below is an example query for getting aDAU and aMAU using `clients_last_seen`.
```

Comment:
```suggestion
[`clients_last_seen`](../datasets/bigquery/clients_last_seen/reference.md) is a more efficient option that includes precalculated aMAU, but is only available in BigQuery. Below is an example query for getting aDAU and aMAU using `clients_last_seen`.
```

Reply:
getting aDAU isn't more efficient in `clients_last_seen`, because `COUNT(*)` in `clients_daily` is v efficient. It is more efficient for aMAU, but sometimes aDAU is enough for people